lib/project_types/script/layers/infrastructure/script_service.rb

```diff
@@ -15,29 +15,44 @@ def push(
           extension_point_type:,
           script_name:,
           script_content:,
-          compiled_type:,
           api_key: nil,
           force: false,
           metadata:,
           config_ui:
         )
-          query_name = "app_script_update_or_create"
+          url = UploadScript.new(ctx).call(api_key, script_content)
+
+          query_name = "app_script_set"
+
+          # TODO: to be properly set https://github.com/Shopify/script-service/pull/3416
```

Comment:
Hey @Thenura, I noticed your change for the following args in https://github.com/Shopify/script-service/pull/3416. Could please help properly send these args in this PR? Thanks!

Reply:
I've removed this as it looks to be resolved.